### DISCUSSION TITLE
Is it possible to change animate the colour of a active piano key when using midi file

### DISCUSSION THREAD

**Discussion by 8bitsboy** [JacquesLucke/animation_nodes#1937] [Q&A]
I created a animation based on a midi file. Rotating the active key of a piano.
I wondered if I could also highlight the active key by animating the shader color for instance?

<img width="791" alt="Screenshot 2023-12-07 at 21 19 37" src="https://github.com/JacquesLucke/animation_nodes/assets/2795951/135794e4-7637-48ca-a4c2-fada057edc95">

Thanks for your help!

Gregory

**Comment by OmarEmaraDev** [comment]
Looks like you are somewhat on the right track. Once you write the color in Animation Nodes, you can access it from the shader using the Object Info node. What are you having difficulty with?
